--- a/Cargo.toml
+++ b/Cargo.toml
@@ -109,7 +109,7 @@
 default-features = false
 
 [dependencies.crypto-bigint]
-version = "0.7.0-rc.28"
+version = "0.7.0"
 features = [
     "zeroize",
     "alloc",
@@ -117,16 +117,16 @@
 default-features = false
 
 [dependencies.crypto-common]
-version = "0.2.0-rc.15"
+version = "0.2.0"
 features = ["getrandom"]
 optional = true
 
 [dependencies.crypto-primes]
-version = "0.7.0-pre.9"
+version = "0.7.0"
 default-features = false
 
 [dependencies.digest]
-version = "0.11.0-rc.11"
+version = "0.11.0"
 features = [
     "alloc",
     "oid",
@@ -143,7 +143,7 @@
 default-features = false
 
 [dependencies.pkcs8]
-version = "0.11.0-rc.10"
+version = "0.11.0"
 features = [
     "alloc",
     "pem",
@@ -166,19 +166,19 @@
 optional = true
 
 [dependencies.sha1]
-version = "0.11.0-rc.5"
+version = "0.11.0"
 features = ["oid"]
 optional = true
 default-features = false
 
 [dependencies.sha2]
-version = "0.11.0-rc.5"
+version = "0.11.0"
 features = ["oid"]
 optional = true
 default-features = false
 
 [dependencies.signature]
-version = "3.0.0-rc.10"
+version = "3.0.0"
 features = [
     "alloc",
     "digest",
@@ -187,7 +187,7 @@
 default-features = false
 
 [dependencies.spki]
-version = "0.8.0-rc.4"
+version = "0.8.0"
 features = ["alloc"]
 optional = true
 default-features = false
@@ -232,12 +232,12 @@
 version = "1.0.89"
 
 [dev-dependencies.sha1]
-version = "0.11.0-rc.5"
+version = "0.11.0"
 features = ["oid"]
 default-features = false
 
 [dev-dependencies.sha2]
-version = "0.11.0-rc.5"
+version = "0.11.0"
 features = ["oid"]
 default-features = false
 
