diff --git a/meson.build b/meson.build
index 3ce12dc8..1973ccaa 100644
--- a/meson.build
+++ b/meson.build
@@ -1194,7 +1194,7 @@ test(
         dependencies: test_dep),
     env: test_env,
 )
-if get_option('enable-x11')
+if false
     has_xvfb = find_program('Xvfb', required: false)
     has_xkbcomp = find_program('xkbcomp', required: false)
     # We only warn because the build machine may not be the same
