diff --git a/utils/uds/Makefile b/utils/uds/Makefile
index f401a9b36178..2d44777b1cb5 100644
--- a/utils/uds/Makefile
+++ b/utils/uds/Makefile
@@ -38,7 +38,11 @@ ifeq ($(findstring clang, $(CC)),clang)
 		-Wno-implicit-const-int-float-conversion \
 		-Wno-language-extension-token
 else
-	WARNS =	-Wcast-align	\
+	ifeq ($(filter riscv64%,$(MAKE_HOST)),)
+	Wcast-align = -Wcast-align
+	endif
+
+	WARNS =	$(Wcast-align)	\
 		-Wcast-qual	\
 		-Wformat=2	\
 		-Wlogical-op
