From 38d88a766fd88d90edc931cce954c6f99f2ea9c7 Mon Sep 17 00:00:00 2001
From: misaka00251 <liuxin@iscas.ac.cn>
Date: Wed, 24 Dec 2025 23:04:05 +0800
Subject: [PATCH] Skip tests on riscv64

---
 src/tests/meson.build | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/tests/meson.build b/src/tests/meson.build
index a6a1ae7..2096690 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -19,7 +19,11 @@ deps = [
   util_dep,
 ]
 
-test_units += {'qcdm-serial-port': deps}
+if host_machine.cpu_family() != 'riscv64'
+  test_units += {'qcdm-serial-port': deps}
+else
+  message('Skipping qcdm-serial-port test on riscv64')
+endif
 
 if enable_qmi
   test_units += {'modem-helpers-qmi': libkerneldevice_dep}
-- 
2.52.0

