diff --git a/Cargo.toml b/Cargo.toml
index 3d489e1..6483993 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,7 +34,7 @@ path = "src/test/main.rs"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
 anyhow = "1.0.93"
-clap = { version = "~4.5.51", features = ["derive", "env"] }
+clap = { version = "4.5.51", features = ["derive", "env"] }
 env_logger = "0.11.8"
 ipnet = { version = "2.11.0", features = ["serde"] }
 iptables = "0.6.0"
