diff --git a/tokenizers/Cargo.toml b/tokenizers/Cargo.toml
index e316d97..57b0ffb 100644
--- a/tokenizers/Cargo.toml
+++ b/tokenizers/Cargo.toml
@@ -83,13 +83,6 @@ http = ["hf-hub"]
 unstable_wasm = ["fancy-regex", "getrandom/wasm_js"]
 rustls-tls = ["hf-hub?/rustls-tls"]
 
-[dev-dependencies]
-criterion = "0.6"
-tempfile = "3.10"
-assert_approx_eq = "1.1"
-tracing = "0.1"
-tracing-subscriber = "0.3.18"
-
 [profile.release]
 lto = "fat"
 
