--- a/Cargo.toml
+++ b/Cargo.toml
@@ -128,11 +128,11 @@
 default-features = false
 
 [dependencies.ed25519]
-version = "3.0.0-rc.4"
+version = "3.0.0"
 default-features = false
 
 [dependencies.keccak]
-version = "0.2.0-rc.1"
+version = "0.2.0"
 optional = true
 default-features = false
 
@@ -147,11 +147,11 @@
 default-features = false
 
 [dependencies.sha2]
-version = "0.11.0-rc.5"
+version = "0.11.0"
 default-features = false
 
 [dependencies.signature]
-version = "3.0.0-rc.10"
+version = "3.0.0"
 optional = true
 default-features = false
 
@@ -168,7 +168,7 @@
 version = "1.0"
 
 [dev-dependencies.blake2]
-version = "0.11.0-rc.5"
+version = "0.11.0"
 
 [dev-dependencies.chacha20]
 version = "0.10.0-rc.10"
