diff -up vision-0.23.0/torchvision/csrc/io/decoder/video_stream.cpp.ffmpeg8 vision-0.23.0/torchvision/csrc/io/decoder/video_stream.cpp
--- vision-0.23.0/torchvision/csrc/io/decoder/video_stream.cpp.ffmpeg8	2025-07-17 23:23:54.000000000 +0200
+++ vision-0.23.0/torchvision/csrc/io/decoder/video_stream.cpp	2025-10-17 10:11:28.763466087 +0200
@@ -122,7 +122,11 @@ int VideoStream::copyFrameBytes(ByteStor
 void VideoStream::setHeader(DecoderHeader* header, bool flush) {
   Stream::setHeader(header, flush);
   if (!flush) { // no frames for video flush
+#if LIBAVUTIL_VERSION_INT >= AV_VERSION_INT(58,7,100)
+    header->keyFrame = (frame_)->flags & AV_FRAME_FLAG_KEY;
+#else
     header->keyFrame = frame_->key_frame;
+#endif
     header->fps = av_q2d(av_guess_frame_rate(
         inputCtx_, inputCtx_->streams[format_.stream], nullptr));
   }
