From cda17c080e1ebbea9bc7797ab9eb685590cc976e Mon Sep 17 00:00:00 2001
From: Sakura286 <sakura286@outlook.com>
Date: Thu, 12 Mar 2026 16:28:58 +0800
Subject: [PATCH] Add HIP detect logic

---
 torchvision/csrc/vision.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/torchvision/csrc/vision.cpp b/torchvision/csrc/vision.cpp
index 806e870..7a62927 100644
--- a/torchvision/csrc/vision.cpp
+++ b/torchvision/csrc/vision.cpp
@@ -21,6 +21,8 @@ namespace vision {
 int64_t cuda_version() {
 #ifdef WITH_CUDA
   return CUDA_VERSION;
+#elif defined(WITH_HIP)
+  return (ROCM_VERSION_MAJOR * 100) + ROCM_VERSION_MINOR;
 #else
   return -1;
 #endif
-- 
2.51.0

