diff --git a/tests/meson.build b/tests/meson.build
index 6a9f4cc..559a1cc 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -10,7 +10,6 @@ test_programs = [
   [['giomm_listmodel'], 'test', ['main.cc'], true],
   [['giomm_memoryinputstream'], 'test', ['main.cc'], true],
   [['giomm_simple'], 'test', ['main.cc'], true],
-  [['giomm_tls_client'], 'test', ['main.cc'], true],
   [['glibmm_base64'], 'test', ['main.cc'], false],
   [['glibmm_binding'], 'test', ['main.cc'], false],
   [['glibmm_bool_arrayhandle'], 'test', ['main.cc'], false],
