--- a/Cargo.toml
+++ b/Cargo.toml
@@ -124,7 +124,7 @@
 harness = false
 
 [dependencies.aead_0_6]
-version = "=0.6.0-rc.10"
+version = "0.6.0"
 package = "aead"
 
 [dependencies.aes]
@@ -175,7 +175,7 @@
 optional = true
 
 [dependencies.crypto-bigint]
-version = "=0.7.0-rc.28"
+version = "0.7.3"
 features = ["alloc"]
 
 [dependencies.ctr]
@@ -216,7 +216,7 @@
 ]
 
 [dependencies.elliptic-curve]
-version = "=0.14.0-rc.28"
+version = "=0.14.0-rc.34"
 features = ["ecdh"]
 
 [dependencies.enum_dispatch]
@@ -267,7 +267,7 @@
 version = "0.7"
 
 [dependencies.ml-kem]
-version = "=0.3.0-rc.1"
+version = "0.3.0"
 
 [dependencies.module-lattice]
 version = "0.2"
@@ -305,10 +305,10 @@
 optional = true
 
 [dependencies.pkcs5]
-version = "=0.8.0-rc.13"
+version = "0.8.0"
 
 [dependencies.pkcs8]
-version = "=0.11.0-rc.11"
+version = "0.11.0"
 features = [
     "encryption",
     "std",
@@ -369,10 +369,10 @@
 version = "0.11.0"
 
 [dependencies.signature]
-version = "3.0.0-rc.10"
+version = "3.0.0"
 
 [dependencies.spki]
-version = "=0.8.0-rc.4"
+version = "0.8.0"
 
 [dependencies.ssh-encoding]
 version = "0.2"
