--- rustc-1.91.1-src/src/tools/miri/Cargo.toml.orig	2025-11-07 11:39:30.000000000 +0800
+++ rustc-1.91.1-src/src/tools/miri/Cargo.toml	2025-12-09 13:38:17.344029697 +0800
@@ -39,7 +39,7 @@ features = ['unprefixed_malloc_on_suppor
 [target.'cfg(unix)'.dependencies]
 libc = "0.2"
 # native-lib dependencies
-libffi = { version = "4.1.1", optional = true }
+libffi = { version = "4.1.1", optional = true, features = ["system"]}
 libloading = { version = "0.8", optional = true }
 serde = { version = "1.0.219", features = ["derive"], optional = true }
 
