diff --git a/tests/Makefile b/tests/Makefile
index 4b1e62a..d25985a 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -3,10 +3,7 @@
 # Peter Jones, 2019-06-18 11:10
 #
 
-TESTS = test.dmpstore.export \
-	test.efivar.export \
-	test.grubenv.var \
-	test.bootorder.var \
+TESTS = test.bootorder.var \
 	test.conin.var \
 	test.efivar.threading \
 	test.parse.db \
@@ -20,7 +17,6 @@ TESTS = test.dmpstore.export \
 
 all: clean $(TESTS)
 
-GRUB_PREFIX ?= grub2
 VALGRIND ?=
 
 V ?= 0
@@ -54,8 +50,6 @@ clean:
 
 test.dmpstore.export:
 	$(quiet)echo testing export to DMPSTORE format
-	$(quiet)$(GRUB_PREFIX)-editenv test.dmpstore.export.result.env create
-	$(quiet)$(GRUB_PREFIX)-editenv test.dmpstore.export.result.env set debug=all,-scripting,-lexer
 	$(quiet)truncate -s 512 test.dmpstore.export.result.env
 	$(quiet)LD_LIBRARY_PATH=$(TOPDIR)/src $(EFIVAR) -n {grub}-GRUB_ENV -f test.dmpstore.export.result.env -D -e test.dmpstore.export.result.var
 	$(quiet)@if grep -q "Do not edit" test.dmpstore.export.result.env ; then \
@@ -68,8 +62,6 @@ test.dmpstore.export:
 
 test.efivar.export:
 	$(quiet)echo testing export to libefivar format
-	$(quiet)$(GRUB_PREFIX)-editenv test.efivar.export.result.env create
-	$(quiet)$(GRUB_PREFIX)-editenv test.efivar.export.result.env set debug=all,-scripting,-lexer
 	$(quiet)truncate -s 512 test.efivar.export.result.env
 	$(quiet)LD_LIBRARY_PATH=$(TOPDIR)/src $(EFIVAR) -n {grub}-GRUB_ENV -f test.efivar.export.result.env -e test.efivar.export.result.var
 	$(quiet)if grep -q "Do not edit" test.efivar.export.result.env ; then \
@@ -81,8 +73,6 @@ test.efivar.export:
 	$(quiet)echo passed
 
 test.grubenv.var:
-	$(quiet)$(GRUB_PREFIX)-editenv test.grubenv.var.result.env create
-	$(quiet)$(GRUB_PREFIX)-editenv test.grubenv.var.result.env set debug=all,-scripting,-lexer
 	$(quiet)truncate -s 512 test.grubenv.var.result.env
 	$(quiet)echo test importing from DMPSTORE and exporting to DMPSTORE and symbolic guid-name validation
 	$(quiet)LD_LIBRARY_PATH=$(TOPDIR)/src $(EFIVAR) -n {grub}-GRUB_ENV -f test.grubenv.var.result.env -D -e test.grubenv.var.0.export.var
