diff --git a/safetensors/Cargo.toml b/safetensors/Cargo.toml
index 24bc150..7d708ae 100644
--- a/safetensors/Cargo.toml
+++ b/safetensors/Cargo.toml
@@ -29,11 +29,6 @@ serde = { version = "1.0", default-features = false, features = [
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
 hashbrown = { version = "0.16", features = ["serde"] }
 
-[dev-dependencies]
-criterion = "0.6"
-memmap2 = "0.9"
-proptest = "1.7"
-
 [features]
 default = ["std"]
 std = ["serde/default", "serde_json/default"]
